CheckYourEligibility-Admin/Views/Check/Outcome/Not_Eligible_LA.cshtml

```diff
@@ -22,10 +22,10 @@
     <p class="govuk-body"><a class="govuk-link" rel="noreferrer noopener" target="_blank" href="~/Home/Guidance">See a complete list of acceptable evidence (opens in new tab)</a>.</p>
 
     <h2 class="govuk-heading-m">If you have not received any evidence</h2>
-    <p class="govuk-body">You can either:</p>
+    <p class="govuk-body">You can either contact:</p>
     <ul class="govuk-list govuk-list--bullet">
-        <li>contact the relevant school to ask them to collect it from the parent or guardian</li>
-        <li>email the Department for Education support desk to request a separate check at ecs.admin@education.gov.uk</li>
+        <li>the relevant school to ask them to collect it from the parent or guardian</li>
+        <li>the <a href="https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-URYbNo1-R1EubHA0gNJis5UOVo3VU1KV0E0QkxTMFQ0RFZQUUZCR0E0WC4u">Department for Education support desk</a> to request a separate check</li>
```

Comment:
If there's a way to not hardcode this value, even if just in appsettings.json I'd prefer that.

Reply:
That is a good suggestion I'll look into that